bug report: using `-D clippy::all` causes compiling failed when rustc-1.48.0

code fstype.push_str(".") in utils/src/fuse.rs:269:9 should correct with fstype.push('.')

Detail message as below:
```
[image-service]# make
# TODO: switch to --out-dir when it moves to stable
# For now we build with separate target directories
cargo build --features=virtiofsd --target-dir target-virtiofsd
    Finished dev [unoptimized + debuginfo] target(s) in 0.09s
cargo clippy --features=virtiofsd --target-dir target-virtiofsd -- -Dclippy::all
    Finished dev [unoptimized + debuginfo] target(s) in 0.08s
cargo build --features=fusedev --target-dir target-fusedev
   Compiling nydus-utils v0.1.0 (/home/RustProjects/image-service/utils)
   Compiling rafs v0.1.0 (/home/RustProjects/image-service/rafs)
   Compiling nydus-api v0.1.0 (/home/RustProjects/image-service/api)
   Compiling nydus-rs v0.1.0 (/home/RustProjects/image-service)
    Finished dev [unoptimized + debuginfo] target(s) in 5.12s
cargo clippy --features=fusedev --target-dir target-fusedev -- -Dclippy::all
    Checking nydus-utils v0.1.0 (/home/RustProjects/image-service/utils)
error: calling `push_str()` using a single-character string literal
   --> utils/src/fuse.rs:269:9
    |
269 |         fstype.push_str(".");
    |         ^^^^^^^^^^^^^^^^^^^^ help: consider using `push` with a character literal: `fstype.push('.')`
    |
    = note: `-D clippy::single-char-push-str` implied by `-D clippy::all`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#single_char_push_str

error: aborting due to previous error

error: could not compile `nydus-utils`

To learn more, run the command again with --verbose.
make: *** [Makefile:17: build-fusedev] Error 101

[image-service]# cargo version
cargo 1.48.0 (65cbdd2dc 2020-10-14)
[ image-service]# rustc --version
rustc 1.48.0 (7eac88abb 2020-11-16)
```

--- a/utils/src/fuse.rs
+++ b/utils/src/fuse.rs
@@ -266,7 +266,7 @@ fn fuse_kern_mount(
     );
     let mut fstype = String::from(FUSE_FSTYPE);
     if !subtype.is_empty() {
-        fstype.push_str(".");
+        fstype.push('.');
         fstype.push_str(subtype);
     }
 

